server/prisma/schema.prisma

```diff
@@ -16,10 +15,10 @@ datasource db {
 model Activity {
   id         String   @id @default(auto()) @map("_id") @db.ObjectId
   name       String
-  price      Decimal
+  price      Float
```

Comment:
I forgot, but did we decide to quantify price as pennies (no decimals)? Otherwise LGTM

Reply:
Then this should be an integer not a `Decimal`